core/metacling/src/TCling.cxx

```diff
@@ -6919,14 +6943,87 @@ const char* TCling::GetClassSharedLibs(const char* cls)
    return 0;
 }
 
+/// This interface returns a list of dependent libraries in the form:
+/// lib libA.so libB.so libC.so. The first library is the library we are
+/// searching dependencies for.
+/// Note: In order to speed up the search, we display the dependencies of the
+/// libraries which are not yet loaded. For instance, if libB.so was loaded
+/// the list would contain: lib libA.so libC.so.
+static std::string GetSharedLibDepsSlow(std::string lib,
+                                        cling::Interpreter *interp,
+                                        bool skipLoadedLibs = true)
```

Comment:
This function is only ever called with `skipLoadedLibs = true`. Can you remove the parameter?

Reply:
The skipLoadedLibs is an optimization to the interface. If one wants the real dependency chain, this interface can help. I am worried that if we remove that parameter we will soon forget the generic purpose of the interface.